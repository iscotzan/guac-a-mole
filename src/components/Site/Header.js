// @flow
import * as React from "react";
import { Link } from "react-router-dom";
// import LetterMorph from "react-letter-morph";
// import { Button, Icon } from "semantic-ui-react";
import { Container, Menu } from "semantic-ui-react";
import LanguageToggle from "../../common/LanguageSelector";
import { Translate } from "react-localize-redux";
// import GuacamoleLogo from "./../Game/Guacamole/GuacamoleLogo/GuacamoleLogo";
const Header = () => {
  return (
    <Menu id="mainMenu" borderless className="mainMenuContainer">
      <Container>
        <Menu.Item>
          <LanguageToggle />
        </Menu.Item>
        <Menu.Item className={"effect-underline"}>
          <Link to={"./"}>
            <Translate id="translations.guacamoleGame">Guacamole</Translate>
          </Link>
        </Menu.Item>
        <Menu.Item className={"effect-underline"}>
          <Link to={"./about"}>
            <Translate id="translations.about">About</Translate>
          </Link>
        </Menu.Item>
      </Container>
    </Menu>
  );
};

export default Header;
