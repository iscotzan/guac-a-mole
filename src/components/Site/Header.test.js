import React from "react";
import Header from "./Header";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

const wrapper = shallow(
  <Header />
);
describe("<Header />", () => {
  it("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  it("renders without crashing", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
