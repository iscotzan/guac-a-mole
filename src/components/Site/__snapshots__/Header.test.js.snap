// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crashing 1`] = `
<Menu
  borderless={true}
  className="mainMenuContainer"
  id="mainMenu"
>
  <Container>
    <MenuItem>
      <LocalizedComponent />
    </MenuItem>
    <MenuItem
      className="effect-underline"
    >
      <Link
        replace={false}
        to="./"
      >
        <Translate
          id="translations.guacamoleGame"
        >
          Guacamole
        </Translate>
      </Link>
    </MenuItem>
    <MenuItem
      className="effect-underline"
    >
      <Link
        replace={false}
        to="./about"
      >
        <Translate
          id="translations.about"
        >
          About
        </Translate>
      </Link>
    </MenuItem>
  </Container>
</Menu>
`;
